Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py

```diff
@@ -0,0 +1,94 @@
+class School:
+    def __init__(self, school_number: int, director: str, secretary: str, number_of_students: int):
+        self.__school_number = school_number
+        self.__director = director
+        self.__secretary = secretary
+        self.__number_of_students = number_of_students
+
+    def school_number(self):
+        return self.__school_number
+
+    @property
+    def director(self):
+        """ Encapsulation"""
+
+        return self.__director.capitalize()
+
+    @property
+    def secretary(self):
+        return self.__secretary
+
+    def number_of_students(self):
+        return self.__number_of_students
+
+    def school_bio(self):
+        return f"School number is: {self.__school_number}\n" \
+               f"Director of the schools is: {self.__director}\n" \
+               f"Secretary of the school is: {self.__secretary}\n" \
+               f"Total number of students in school is: {self.__number_of_students}\n"
+
+
+class Classes:
+    """ Polymorphism"""
```

Comment:
Could you please write comments where
Inheritance, # hiding, # incapsulation, how it is written in homework. In your homework, I can't see that you understand this principles

Reply:
Done.